lib/middleware/schemas/settings.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "id": "https://forms-formulaires.alpha.canada.ca/",
+  "type": "object",
+  "title": "GC Forms Settings Schema",
+  "description": "A schema to define a Settings configuration.",
+  "default": {},
+  "examples": [
+    {
+      "internalId": "",
+      "nameEn": "",
+      "nameFr": "",
+      "descriptionEn": "",
+      "descriptionFr": "",
+      "value": ""
+    }
+  ],
+  "properties": {
+    "internalId": {
+      "description": "The internal identified of the Setting",
+      "id": "#/properties/internalId",
+      "type": "string"
+    },
+    "nameEn": {
+      "description": "The displayed name of the Setting to the user in English",
+      "id": "#/properties/nameEn",
+      "type": "string"
+    },
+    "nameFr": {
+      "description": "The displayed name of the Setting to the user in French",
+      "id": "#/properties/nameFr",
+      "type": "string"
+    },
+    "descriptionEn": {
+      "description": "The displayed description of the Setting to the user in French",
+      "id": "#/properties/descriptionEn",
+      "type": ["string", "null"]
+    },
+    "descriptionFr": {
+      "description": "The displayed description of the Setting to the user in French",
+      "id": "#/properties/descriptionFr",
+      "type": ["string", "null"]
+    },
+    "value": {
+      "description": "The value of the Setting in string format",
+      "id": "#/properties/value",
+      "type": ["string", "null"]
+    }
+  },
+  "required": ["internalId", "nameEn", "nameFr"],
```

Comment:
Right now the Prisma schema has the `value` as an optional property.  This is because when we seed the DB we don't know the value of the settings, for example a form ID that doesn't yet exist.  I figured I would let the API handle the same expectations as the DB and allow for a null value.  This also allows an admin user to remove the value temporarily while not deleting the whole setting.